src/example-types/terms_stats.js

```diff
@@ -19,16 +33,18 @@ module.exports = {
           key_data: {
             field,
             size: context.size || 10,
-            order: {
-              // Disable nested path checking for now as we don't need it anymore:
-              //  key +
-              //      may need >inner>inner. when adding additional inner for nested filters per old code
-              //      xor paths - if both are nested or both not, the inner one wot be in inner
-              //      ((!getNestedPath(context.value_field) != !getNestedPath(context.key_field)) ? '>inner.' : '.') +
-              //  order
-              [`twoLevelAgg.${context.order || 'sum'}`]:
-                context.sortDir || 'desc',
-            },
+            order: hasValidMetrics(context)
+              ? orderPaths[context.order]
+              : {
+                  // Disable nested path checking for now as we don't need it anymore:
+                  //  key +
+                  //      may need >inner>inner. when adding additional inner for nested filters per old code
+                  //      xor paths - if both are nested or both not, the inner one wot be in inner
+                  //      ((!getNestedPath(context.value_field) != !getNestedPath(context.key_field)) ? '>inner.' : '.') +
+                  //  order
+                  [`twoLevelAgg.${context.order || 'sum'}`]:
```

Comment:
to clarify what I meant above, there would be a _slight_ change - you'd just either do `twoLevelAgg.${context.order}` or `twoLevelAgg_${context.order}`.value depending on whether it's multi-value metric or not - which would just be if include is specified.

so just something like this:
```js
let orderField = ({ include, order = 'sum' }) =>
  include ? `twoLevelAgg_${order}.value` : `twoLevelAgg.${order}`
```
and then later:
```js
order: {
  [orderField(context)]: context.sortDir || 'desc'
},
```

Reply:
btw, while we're here, we should rename `context` to `node`. The name `context` is a legacy remnant from when these were called `DataContexts` instead of just nodes on a tree

Reply:
Also, you also don't need to make `orderField` a util, it can totally live inline. I just pulled it out to make sure the logic was clearer in this comment